pkg/prove/proof.go

```diff
@@ -42,8 +41,10 @@ func TxInclusion(codec rsmt2d.Codec, data types.Data, origSquareSize, txIndex ui
 	var rowRoots []tmbytes.HexBytes //nolint:prealloc // rarely will this contain more than a single root
 	for i, row := range rowShares {
 		// create an nmt to use to generate a proof
-		tree := wrapper.NewErasuredNamespacedMerkleTree(uint64(origSquareSize))
+		fmt.Println("square size", data.OriginalSquareSize, len(row), "rows", startRow, endRow, i, "leaves", startLeaf, endLeaf, "positions", startPos, endPos, "txindex", txIndex)
+		tree := wrapper.NewErasuredNamespacedMerkleTree(data.OriginalSquareSize)
 		for j, share := range row {
+			fmt.Println(j, share[:25])
```

Comment:
Looks like you forgot some debug output.

Reply:
:man_facepalming: